src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -40,6 +41,41 @@ public Task Head()
             }
         }
 
+        [RavenAction("/databases/*/docs/$", "GET")]
```

Comment:
The problem is that we have an endpoint /docs?id=orders/1
but this endpoint returns always an array because you can specify more than one 'id'

This was bothering us here, since we know that in 3.x (despite having /docs?id= endpoint) we had /docs/{*id} one to easily (e.g. in browser) show single document